docs/intro_wavefunction.rst

```diff
@@ -873,6 +873,9 @@ Additional information:
 - ``algorithm`` algorithms used in multi-Slater determinant implementation. ``table_method`` table method of Clark et al. :cite:`Clark2011` .
   ``precomputed_table_method`` adds partial sum precomputation on top of ``table_method``.
 
+- When the determinants are read from an HDF5 file in the ``detlist`` child, the hdf5 representation of the multideterminant wavefunction must use unsigned integers to represent the determinants. The ``utils/determinants_tools.py`` script described in further detail below will check that the determinants are stored using the correct type.
```

Comment:
do the ints need to be 64-bit unsigned, or is the converter flexible enough to handle other sizes?

Reply:
I am not sure about other paths into the code (xml), but at least through HDF5 you need 64 bit.  I've fixed the docs as well